src/cogito/package.d

```diff
@@ -47,7 +71,7 @@ private Result runOnFile(string file)
 auto runOnFiles(R)(R args)
 if (isInputRange!R && is(ElementType!R == string))
 {
-    return args.map!runOnFile();
+    return args.map!(toFiles).joiner.array.sort.map!(runOnFile)();
```

Comment:
Just a matter of taste. We would have used fewer parentheses but more line breaks:
```
return args
    .map!toFiles
    .joiner
    .array
    .sort
    .map!runOnFile;
```